share/functions/__fish_print_ninja_targets.fish

```diff
@@ -0,0 +1,5 @@
+function __fish_print_ninja_targets
+    if [ -f build.ninja ]
+        ninja -t targets | sed 's/:.*//'
```

Comment:
What happens here when you call ninja without that file?

If it's just some error message, it's probably easier to suppress it so this can be used even if there's some other filename that ninja can use (now or in the future).